src/auth/verifyingPopUpLoginSAS9.ts

```diff
@@ -0,0 +1,18 @@
+import { delay } from '../utils'
+
+export async function verifyingPopUpLoginSAS9(loginPopup: Window) {
```

Comment:
I'd probably call this file and function `verifySas9Login`.

Reply:
fixed.